appbuilder/services/livesync/ios-livesync-service.ts

```diff
@@ -0,0 +1,74 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import iOSProxyServices = require("../../../mobile/ios/device/ios-proxy-services");
+import * as path from "path";
+import * as shell from "shelljs";
+let osenv = require("osenv");
+
+export class IOSLiveSyncService implements IPlatformLiveSyncService {
+	private get $project(): any {
+		return this.$injector.resolve("project");
+	}
+
+	private get $projectConstants(): IProjectConstants {
+		return this.$injector.resolve("projectConstants");
+	}
+
+	constructor(private _device: Mobile.IDevice,
```

Comment:
Same as the constructor in `AppBuilderAndroidLiveSyncService` - can you declare this as 
```TypeScript
private _device: Mobile.IiOSDevice
```

and just use it rather than forming a `get` property or casting it